Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/StackGrabbingTest.java

```diff
@@ -1,4 +1,4 @@
-package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+/*package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
```

Comment:
Why is this whole file commented out? If you don't want it, disable it or delete it.